control_panel_api/permissions.py

```diff
@@ -31,23 +28,23 @@ class S3BucketPermissions(IsSuperuser):
     pass
 
 
-class IsSelf(BasePermission):
+class UserPermissions(BasePermission):
+    """
+    Superusers can do anything, normal users can only access themselves,
+    unauthenticated users cannot do anything
+    """
 
     def has_permission(self, request, view):
-        if super().has_permission(request, view):
+        if is_superuser(request.user):
             return True
 
-        if hasattr(request.user, 'auth0_id'):
-            return view.kwargs.get('pk') == request.user.auth0_id
+        if request.user.is_anonymous():
+            return False
 
-        return False
+        return view.action not in ('create', 'destroy', 'list')
```

Comment:
Nice that you can check the `view.action`, much nicer than checking the `request.method` 👍